### PR TITLE
Change logs / chains folder name & location + some code refactoring 

### PR DESCRIPTION
**Pull request by platfowner**
Summary:
- Remove an obsolete auth.fid use case from getOwnerPermissions()
- Separate 'logs' folder from code by moving it to upper directory
- Rename 'blockchains' folder -> 'chains'
- Change block file suffix to 'json.gz' 
- Rename renameVersion() -> transferStateTree()
- Uncomment some fixed integration test cases
- Update .gitignore file
- Tweak levels of some logging messages 
- Remove some obsolete TODOs

### CODE REVIEW

File: node/index.js [hunk 1]
```diff
@@ -154,7 +154,7 @@ class BlockchainNode {
     }
     if (FeatureFlags.enableVersionRenaming) {
```
Comment (liayoo) [inline node/index.js:155]: suggestion) rename the flag & update the logging below?

File: start_servers_afan.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-rm -rf blockchain/blockchains logger/log
+rm -rf ./chains/ ./log/
```
Comment (liayoo) [inline start_servers_afan.sh:1]: logs?

**Review by liayoo — APPROVED** [review]
LGTM with some comments, thanks! Nice to see the folder reorg 👍

File: start_servers_afan.sh (hunk 2 above)
Comment (platfowner) [reply start_servers_afan.sh]: Just fixed. Thanks!

File: node/index.js (hunk 1 above)
Comment (platfowner) [reply node/index.js]: Right, it should be updated. Done, thanks.

**Comment by platfowner** [comment]
All comments addressed. Thanks for the review. Now merging..

**Pull request merged**